packages/react/src/__tests__/ReactStrictMode-test.js

```diff
@@ -75,7 +77,7 @@ describe('ReactStrictMode', () => {
   });
 
   // @gate __DEV__
-  it('should invoke precommit lifecycle methods twice', () => {
+  it('should invoke only precommit lifecycle methods twice in legacy roots', async () => {
```

Comment:
Double invoking commit lifecycle methods in new roots is tested in StrictEffects-test which also tests not double invoking commit lifecycles in legacy roots. So here we really only need to focus on precommit lifecycle